src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -121,6 +121,12 @@ public DashboardWebApplication(Action<WebApplicationBuilder>? configureBuilder =
         // Add services to the container.
         builder.Services.AddRazorComponents().AddInteractiveServerComponents();
         builder.Services.AddCascadingAuthenticationState();
+        builder.Services.AddResponseCompression(options =>
+        {
+            options.EnableForHttps = true;
+            // Avoid compressing HTML to mitigate user supplied data being compressed over HTTPS
```

Comment:
Should the comment say because of side channel attack? Or maybe that's obvious.

Reply:
Maybe just link to [our docs about it](https://learn.microsoft.com/aspnet/core/performance/response-compression?view=aspnetcore-8.0#compression-with-https)?

Reply:
Seems reasonable

Reply:
What are the rules when it comes to Blazor compression? Are they different from regular server rendered HTML?

We only want static resources compressed:
* JavaScript
* CSS
* Images
* SVG

Everything else should be excluded:
* Communication over SignalR websockets
* gRPC calls (remember the dashboard has some gRPC endpoints)
* Re-rendered HTML (I think that's disabled)

Reply:
I'd feel safer with an allow list than a deny list.

Reply:
@JamesNK changed it to use an allow list